pkgs/development/libraries/opencascade-occt/default.nix

```
@@ -1,20 +1,44 @@
-{ lib, stdenv, fetchurl, fetchpatch, cmake, ninja, tcl, tk,
-  libGL, libGLU, libXext, libXmu, libXi, darwin }:
+{ lib
+, stdenv
+, fetchurl
+, fetchpatch
+, cmake
+, ninja
+, tcl
+, tk
+, libGL
+, libGLU
+, libXext
+, libXmu
+, libXi
+, darwin
+}:
 
 stdenv.mkDerivation rec {
   pname = "opencascade-occt";
-  version = "7.6.2";
+  version = "7.8.1";
   commit = "V${builtins.replaceStrings ["."] ["_"] version}";
 
   src = fetchurl {
     name = "occt-${commit}.tar.gz";
     url = "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=${commit};sf=tgz";
```

```suggestion

  src = let
    commit = "V${builtins.replaceStrings ["."] ["_"] version}";
  in fetchurl {
    name = "occt-${commit}.tar.gz";
    url = "https://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=${commit};sf=tgz";
```
so that we do not inject it as an env. or maybe a level higher, not that important